Cabal/Cabal.cabal

```diff
@@ -96,7 +96,6 @@ library
     Distribution.Utils.MD5
     Distribution.Utils.String
     Distribution.Utils.Structured
-    Distribution.Compat.CreatePipe
```

Comment:
We could keep `Distribution.Compat.CreatePipe` as a trivial reexport to avoid requiring library users to make changes here if they were using the module. And/or I guess this module going away should be documented explicitly in the Changelog?

Reply:
If it's so cheap, I'd say let's keep the compat. The PR breaks (legally) enough stuff.